### PR TITLE
Added a fitter_kwargs argument to k_fold_cross_validation utility function

### PR DESCRIPTION
**Pull request by cosmicBboy**
- when doing k_fold_cross_validation, the function should be 
able to pass in key-word arguments to the fitter
- for example, for the coxph model, you should be able to 
pass in the strata argument
- updated k_fold_cross_validation docstring

### CODE REVIEW

File: lifelines/plotting.py [hunk 1]
```diff
@@ -207,7 +207,6 @@ def plot_lifetimes(lifetimes, event_observed=None, birthtimes=None,
         plt.scatter((birthtimes[i]) + lifetimes[i], N - 1 - i, color=c, s=30, marker=m)
 
     plt.ylim(-0.5, N)
-    plt.show(block=block)
```
Comment (CamDavidsonPilon) [inline lifelines/plotting.py]: is this a necessary change?

File: lifelines/utils/__init__.py [hunk 2]
```diff
@@ -482,7 +484,8 @@ def k_fold_cross_validation(fitters, df, duration_col, event_col=None,
 
         for fitter, scores in zip(fitters, fitterscores):
             # fit the fitter to the training data
-            fitter.fit(training_data, duration_col=duration_col, event_col=event_col)
+            fitter.fit(training_data, duration_col=duration_col,
+                       event_col=event_col, **fitter_kwargs)
```
Comment (CamDavidsonPilon) [inline lifelines/utils/__init__.py]: :+1:

**Comment by CamDavidsonPilon** [comment]
Cool, just a single question, otherwise lgtm!

**Comment by cosmicBboy** [comment]
Oops! It's not relevant to the revision, I'll revise

**Comment by CamDavidsonPilon** [comment]
Thanks for the contribution @cosmicBboy! Merging now

**Comment by cosmicBboy** [comment]
Awesome, thanks for developing this! survival modeling in Python is a godsend :)

**Pull request merged**